homeassistant/components/zwave_me/__init__.py

```diff
@@ -60,26 +64,35 @@ async def async_establish_connection(self):
         is_connected = await self.zwave_api.get_connection()
         return is_connected
 
-    def add_device(self, device: ZWaveMeData) -> None:
-        """Send signal to create device."""
-        if device.deviceType in ZWAVE_ME_PLATFORMS and self.platforms_inited:
-            if device.id in self.device_ids:
-                dispatcher_send(self._hass, f"ZWAVE_ME_INFO_{device.id}", device)
+    def add_entity(self, entity: ZWaveMeData) -> None:
```

Comment:
Please don't rename these methods and parameters. The parameter represents a new device from the api library, not an entity in Home Assistant.